src/filters/truncate.js

```diff
@@ -3,7 +3,7 @@
 angular.module('openshiftCommonUI')
   .filter('truncate', function() {
     return function(str, charLimit, useWordBoundary, newlineLimit) {
-      if (!str) {
+      if (typeof str !== 'string' || typeof charLimit !== 'number') {
```

Comment:
This will ignore `newlineLimit` if `charLimit` isn't set. Not sure if we ever actually do that, but it might be useful with a one line limit and `text-overflow: ellipsis;` where we use CSS to truncate a long, single line.